ProjectEureka/project-eureka-flutter
# PE-19 | Implement First Time Onboarding

**What I did:**

- Created First time onboarding page

**How to test:**

- Clone branch
- swipe right/left or click small dots to see different slides
- test skip button
- test got it button in the end
- upon clicking got it - you should see placeholder page
